addons/mail/static/src/discuss/call/common/rtc_session_model.js

```diff
@@ -121,6 +121,7 @@ export class RtcSession extends Record {
             }
         },
     });
+    promoteFullscreen = fields.Attr(false);
```

Comment:
I would not move the field to `rtc_session_model`, mostly because removal of the other field doesn't follow stable policy.

But you can add a `onAdd` / `onDelete` / `onUpdate` on `selfSession = fields.One()` to have it change the related `channel.promoteFullscreen` value.

And ok to deprecate old code of promote if this is necessary, but I think we can keep `CALL_PROMOTE_FULLSCREEN` even if some values are deprecated. The less code we deprecate the better, since this results to smaller diff (thus safer change) in stable.